### PR TITLE
feat: made the courses as read-only if old mongo store

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[2U-707](https://youtrack.raccoongang.com/issue/2U-707)

### CODE REVIEW

File: src/studio-home/card-item/index.jsx [hunk 1]
```diff
@@ -19,23 +19,27 @@ const CardItem = ({
   } = useSelector(getStudioHomeData);
   const courseUrl = new URL(url, getConfig().STUDIO_BASE_URL);
   const subtitle = isLibraries ? `${org} / ${number}` : `${org} / ${number} / ${run}`;
+  const readOnlyItem = !lmsLink && !rerunLink && !url;
```
Comment (PKulkoRaccoonGang) [inline src/studio-home/card-item/index.jsx:22]: Maybe looks better

```suggestion
  const readOnlyItem = !(lmsLink || rerunLink || url);
```

Comment (PKulkoRaccoonGang) [reply src/studio-home/card-item/index.jsx:22]: Similar suggestion for the line below

**Review by PKulkoRaccoonGang — APPROVED** [review]

**Review by vladislavkeblysh — APPROVED** [review]

File: src/studio-home/card-item/index.jsx (hunk 1 above)
Comment (ruzniaievdm) [reply src/studio-home/card-item/index.jsx:22]: sure, we can do in this way, thanks!

**Review by wowkalucky — APPROVED** [review]
OK..

**Pull request merged**